packages/shared/lib/core/wallet/classes/activity.class.ts

```diff
@@ -208,11 +212,14 @@ export class Activity implements IActivity {
             const days = Math.floor(elapsedTime / (1000 * 60 * 60 * 24))
             const hours = Math.floor((elapsedTime / (1000 * 60 * 60)) % 24)
             const minutes = Math.floor((elapsedTime / (1000 * 60)) % 60)
+            const seconds = Math.floor((elapsedTime / 1000) % 60)
```

Comment:
We have some constants in the `@lib/time` module that should be used here, like `MILLISECONDS_PER_SECOND` and `MINUTES_PER_SECOND`; they help get rid of magic constants.